### PR TITLE
[doc] Improve Pulsar Administration Upgrade 

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Administration document (Administration-upgrade section): https://pulsar.apache.org/docs/en/administration-upgrade/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
@Anonymitaet please review~

File: site2/docs/administration-upgrade.md [hunk 1]
```diff
@@ -36,20 +34,20 @@ To upgrade an Apache Pulsar cluster, follow the upgrade sequence.
        ```shell
        bin/bookkeeper shell autorecovery -disable
        ```  
-    - b. Rollout the upgraded version to all bookies in the cluster after you have determined a version is safe after canary.  
-    - c. After all bookies are upgraded, re-enable `autorecovery` with the following command.
+    - b. Rollout the upgraded version to all bookies in the cluster after you determines that a version is safe after canary.
```
Comment (Anonymitaet) [inline site2/docs/administration-upgrade.md]:
```suggestion
    - b. Rollout the upgraded version to all bookies in the cluster after you determine that a version is safe after canary.  
```

File: site2/docs/administration-upgrade.md [hunk 2]
```diff
@@ -98,7 +96,7 @@ To upgrade bookie to a new version, complete the following steps:
 
 #### Canary rollback
 
-If issues occur during canary test, you can shut down the problematic bookie node. Other bookies in the cluster will replace this problematic bookie node with autorecovery. 
+If issues occur during canary test, you can shut down the problematic bookie node. Other bookies in the cluster replaces this problematic bookie node with autorecovery.
```
Comment (Anonymitaet) [inline site2/docs/administration-upgrade.md]:
```suggestion
If issues occur during the canary test, you can shut down the problematic bookie node. Other bookies in the cluster replaces this problematic bookie node with autorecovery. 
```

File: site2/docs/administration-upgrade.md [hunk 3]
```diff
@@ -108,18 +106,18 @@ Before upgrading, you have to decide whether to upgrade the whole cluster at onc
...
-While upgrading in both scenarios, the procedure is the same for each bookie.
+While you upgrade in both scenarios, the procedure is the same for each bookie.
 
 1. Stop the bookie. 
 2. Upgrade the software (either new binary or new configuration files).
 2. Start the bookie.
 
 > **Advanced operations**   
-> When upgrading a large BookKeeper cluster in rolling upgrade scenario, it is slow to upgrade one bookie at a time. If you have configured rack-aware or region-aware placement policy, you can upgrade bookies rack by rack or region by region. It speeds up the whole upgrade process.
+> When you upgrade a large BookKeeper cluster in rolling upgrade scenario, upgrading one bookie at a time is slow. If you configure rack-aware or region-aware placement policy, you can upgrade bookies rack by rack or region by region, which speeds up the whole upgrade process.
```
Comment (Anonymitaet) [inline site2/docs/administration-upgrade.md]:
```suggestion
> When you upgrade a large BookKeeper cluster in a rolling upgrade scenario, upgrading one bookie at a time is slow. If you configure rack-aware or region-aware placement policy, you can upgrade bookies rack by rack or region by region, which speeds up the whole upgrade process.
```

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run integration tests
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Comment by Monica-zy** [comment]
run java8 tests

**Pull request merged**